easybuild/easyblocks/g/gromacs.py

```diff
@@ -208,6 +209,15 @@ def configure_step(self):
             # version of GROMACS. Just prepare first part of cmd here
             plumed_cmd = "plumed-patch -p -e %s" % engine
 
+        # Ensure that the GROMACS log files report how the code was patched
+        # during the build, so that any problems are easier to diagnose.
+        if '-DGMX_VERSION_STRING_OF_FORK=' not in self.cfg['configopts']:
+            if plumed_root:
+                gromacs_version_string_suffix = 'PLUMED-%s-%s' % (get_software_version('PLUMED'), VERBOSE_VERSION)
+            else:
+                gromacs_version_string_suffix = VERBOSE_VERSION
```

Comment:
Just noticed this while testing: `VERBOSE_VERSION` doesn't include `EasyBuild` at all (it's verbose because it may include a Git commit ID, if EasyBuild is being run straight from a Git repository).

So you probably want something like this instead:

```python
gromacs_version_string_suffix = 'EasyBuild-%s' % VERBOSE_VERSION
if plumed_root:
    gromacs_version_string_suffix += '-PLUMED-%s' + get_software_version('PLUMED')

Reply:
Done